index.dd

```diff
@@ -529,7 +529,7 @@ void main()
     // Code is executed in any case upon scope exit.
     scope(exit) { writeln("Exiting main."); }
     // File is closed deterministically at scope's end.
-    foreach (line; File("text.txt").byLine())
+    foreach (line; File("/etc/issue").byLine())
```

Comment:
`/etc/...` is not very portable (at least not on Windows), so how about:
```suggestion
    foreach (line; File(__FILE_FULL_PATH__).byLine())
```

Reply:
Much better, I'll change it